tests/org.jboss.tools.forge.ui.bot.test/src/org/jboss/tools/forge2/ui/bot/wizard/test/DeployScaffoldDBTest.java

```diff
@@ -48,6 +48,8 @@
 	private List<String> tableNames = new ArrayList<String>();
 
 	private static final String PACKAGE = GROUPID + ".model";
+	
+	private static final String SERVER_NAME = "WildFly 10.x Server";
```

Comment:
And one last thing I just noticed - you should not assume that RedDeer will always create a server with this name. Correct way of obtaining sever name is as follows:

Inject requirement:
`@InjectRequirement
    private ServerRequirement sr;
`

and later in your code:
`sr.getServerNameLabelText(sr.getConfig())`